### PR TITLE
WIP: Mixedarch 393

### PR DESCRIPTION
**Pull request by zniu1011**

### CODE REVIEW

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-baremetal-compact-agent-ipv4-static-day2-f14

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-baremetal-compact-agent-ipv4-static-day2-f14

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-baremetal-compact-agent-ipv4-static-day2-f14

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-baremetal-compact-agent-ipv4-static-day2-f14

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-baremetal-compact-agent-ipv4-static-day2-f14

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-baremetal-compact-agent-ipv4-static-day2-f14

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-baremetal-compact-agent-ipv4-static-day2-f14

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-baremetal-compact-agent-ipv4-static-day2-f14

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-baremetal-compact-agent-ipv4-static-day2-f14

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-baremetal-compact-agent-ipv4-static-day2-f14

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-baremetal-compact-agent-ipv4-static-day2-f14

**Comment by lwan-wanglin** [comment]
if should we add profile to `openshift-openshift-tests-private-release-4.15__multi-nightly.yaml` file?

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-baremetal-compact-agent-ipv4-static-day2-f14

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-baremetal-compact-agent-ipv4-static-day2-f14

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-baremetal-compact-agent-ipv4-static-day2-f14

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-baremetal-compact-agent-ipv4-static-day2-f14

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-baremetal-compact-agent-ipv4-static-day2-f14

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-baremetal-compact-agent-ipv4-static-day2-f14

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-baremetal-compact-agent-ipv4-static-day2-f14

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-baremetal-compact-agent-ipv4-static-day2-f14

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-baremetal-compact-agent-ipv4-static-day2-f14

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-baremetal-compact-agent-ipv4-static-day2-f14

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-baremetal-compact-agent-ipv4-static-day2-f14

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-baremetal-compact-agent-ipv4-static-day2-f14

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-baremetal-compact-agent-ipv4-static-day2-f14

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-baremetal-compact-agent-ipv4-static-day2-f14

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-baremetal-compact-agent-ipv4-static-day2-f14

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-baremetal-compact-agent-ipv4-static-day2-f14

**Comment by zniu1011** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-baremetal-compact-agent-ipv4-static-day2-f14

File: ci-operator/step-registry/baremetal/lab/agent/install/baremetal-lab-agent-install-commands.sh [hunk 1]
```diff
@@ -248,3 +289,37 @@ if ! wait "$!"; then
...
+    exit 0
+fi
+
+oc extract -n openshift-machine-api secret/worker-user-data-managed --keys=userData --to=- > "${SHARED_DIR}"/worker.ign
+
+echo -e "\nCopying ignition files into bastion host..."
+chmod 644 "${SHARED_DIR}"/*.ign
+scp "${SSHOPTS[@]}" "${SHARED_DIR}"/*.ign "root@${AUX_HOST}:/opt/html/${CLUSTER_NAME}/"
+
+for bmhost in $(yq e -o=j -I=0 '.[]' "${SHARED_DIR}/hosts.yaml"); do
```
Comment (aleskandro) [inline ci-operator/step-registry/baremetal/lab/agent/install/baremetal-lab-agent-install-commands.sh:304]: Should this logic go into the upi/baremetal/heterogeneous step directly? All the scale-out logic is expected there and you'd need less stuff to do there for it.

Comment (zniu1011) [reply ci-operator/step-registry/baremetal/lab/agent/install/baremetal-lab-agent-install-commands.sh:304]: Thanks @aleskandro , I will update. Please move to https://github.com/openshift/release/pull/47637 for the updates later.

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-baremetal-compact-agent-ipv4-static-day2-f14

**Pull request closed without merging**